vtex/faststore
# Feat: Next 13 - Home - BannerText

## What's the purpose of this pull request?

Migration of the `BannerText` section from the pages router to the app router (Next 13).

## How it works?

![Screenshot 2024-08-14 at 10 07 31](https://github.com/user-attachments/assets/850c9d89-227c-4159-890b-ff325744ec04)

## How to test it?

1. Run `yarn dev`
2. Make sure that all the current pages are working as expected (`/`, `/office` and some `/<slug>/p`);
3. Make sure `/fs-next-update` is rendering with the `BannerText` section without errors.

### Starters Deploy Preview





## References

- [Jira task](https://vtex-dev.atlassian.net/browse/SFS-1203)
